packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -117,6 +123,7 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
             scrollToFn: elementScroll,
             observeElementOffset,
             observeElementRect,
+            isScrollingResetDelay: 0,
```

Comment:
This is basically a debounce duration ([defaults to 150 ms](https://tanstack.com/virtual/v3/docs/api/virtualizer#isscrollingresetdelay)) that controls how long after the last scroll `isScrolling` gets set to `false`. A delay of zero should make it feel more responsive (assuming the virtualizer's change handler isn't otherwise debounced/delayed).

Reply:
We were already setting `scrollingDelay` (which was renamed to `isScrollingResetDelay`) to 5. Might as well be 0, though.

Reply:
Changed it to 250ms (which is also the default) to avoid the noisy flickering that Jesse reported in another thread.